### ISSUE TITLE
ImportError in Python 3.10: cannot import name 'MutableMapping' from 'collections'

### ISSUE THREAD

**Issue by B34n2** [twoolie/NBT#154]
![image](https://user-images.githubusercontent.com/92004065/136130967-53f1633b-b689-414d-9d81-4e4fb391a6f0.png)

**Comment by macfreek** [comment]
I'm reopening this issue. We should support Python 3.10.

**Comment by Fenixin** [comment]
Just tested the branch in regionfixer with all the worlds (new and old) that I've got using python 3.10 and it seems to work for me.

**Comment by FolfyBlue** [comment]
It seems to be working, will this be pushed into the main branch?

**Comment by JakobDev** [comment]
I can also confirm that the fix-python-3.10 branch is working. So It's time for a update on PyPI.

**Comment by Synergetic00** [comment]
Bash shell script to fix errors, since people aren't merging yet..

```bash
find $LOCALAPPDATA\\Programs\\Python\\Python310\\Lib\\site-packages\\nbt -name '*.py' -type f -print0 | xargs -0 sed -i 's|from collections |from collections.abc |g'
```

**Comment by macfreek** [comment]
Fix is merged. It is also pushed as a bug-fix (1.5.1) to PyPi.

@einekratzekatze Thanks for raising this issue and proposing a fix!
@Synergetic00 Thanks for stepping it up a notch, and providing a work-around while we were lagging with the merge.
@FolfyBlue, @JakobDev thanks for taking the time to test the fix.

If any of you is interested of continuing to support NBT, please speak to @twoolie. I know both he and I would be thrilled with some more knowledgable eyes on the code (since neither of us is active Minecraft players at the moment).
